From 028cbc56d41c81d5f8ab272dd97ebb679e64970f Mon Sep 17 00:00:00 2001
From: ptrcnull <git@ptrcnull.me>
Date: Sat, 20 Dec 2025 23:53:04 +0100
Subject: [PATCH] musl-no-linux-prctl.patch

see https://www.openwall.com/lists/musl/2025/06/12/11

    /usr/include/sys/prctl.h:88:8: error: redefinition of 'prctl_mm_map'
       88 | struct prctl_mm_map {
          |        ^
    /usr/include/linux/prctl.h:134:8: note: previous definition is here
      134 | struct prctl_mm_map {
          |        ^
---
 third_party/libwebrtc/rtc_base/platform_thread_types.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/third_party/libwebrtc/rtc_base/platform_thread_types.cc b/third_party/libwebrtc/rtc_base/platform_thread_types.cc
index 697bedadb944f..3da0a9e0a593b 100644
--- a/third_party/libwebrtc/rtc_base/platform_thread_types.cc
+++ b/third_party/libwebrtc/rtc_base/platform_thread_types.cc
@@ -12,7 +12,6 @@
 
 // IWYU pragma: begin_keep
 #if defined(WEBRTC_LINUX)
-#include <linux/prctl.h>
 #include <sys/prctl.h>
 #include <sys/syscall.h>
 
-- 
2.56.0

